be/src/runtime/runtime_state.h

```diff
@@ -549,9 +529,6 @@ class RuntimeState {
     // Lock protecting _error_log and _unreported_error_idx
     std::mutex _error_log_lock;
```

Comment:
Can this lock be removed too?

Reply:
No, this can't be remove, it will be used to write error log of load to disk

Reply:
```
void RuntimeState::append_error_msg_to_file(const std::string& line, const std::string& error_msg, bool is_summary) {
    std::lock_guard<std::mutex> l(_error_log_lock);
    if (_query_options.query_type != TQueryType::LOAD) {
        return;
    }
    // If file havn't been opened, open it here
    if (_error_log_file == nullptr) {
        Status status = create_error_log_file();
        if (!status.ok()) {
            LOG(WARNING) << "Create error file log failed. because: " << status.message();
            if (_error_log_file != nullptr) {
                _error_log_file->close();
                delete _error_log_file;
                _error_log_file = nullptr;
            }
            return;
        }
    }
```